.github/workflows/check_changed_files.yml

```diff
@@ -0,0 +1,51 @@
+name: Check changed files
+
+on: 
+    pull_request:
+        branches:
+            - main
+
+jobs:
+
+    check-changed-files:
+
+        runs-on: ubuntu-latest
+
+        env:
+            TEMPLATE_REPO: neurodatascience/nipoppy
+            PATH_CURRENT_REPO: repo_to_check
+            PATH_TEMPLATE_REPO: template_repo
+
+        if: github.repository_owner == 'neurodatascience' && github.event.repository.name != 'nipoppy'
```

Comment:
are we matching exact repo name string or anything that begins with "nipoppy"?

Reply:
Matching the exact repo string: I tested in a dummy repo (`testpoppy`) and the workflow is skipped in [testpoppy](https://github.com/neurodatascience/testpoppy/actions) but runs on [testpoppy-ppmi](https://github.com/neurodatascience/testpoppy-ppmi/actions)